OrdersApplication/OrdersViewController.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ViewController.swift
+//  OrdersApplication
+//
+//  Created by Esraa Khaled   on 22/09/2022.
+//
+
+import UIKit
+
+class OrdersViewController: UIViewController {
+    
+    @IBOutlet weak var tableView: UITableView!{
+        didSet{
+            tableView.delegate = self
+            tableView.dataSource = self
```

Comment:
Add to configure table view instead of this.

Reply:
solved